diff --git a/font-src/glyphs/number/4.ptl b/font-src/glyphs/number/4.ptl
index 60cfbff..18e50cd 100644
--- a/font-src/glyphs/number/4.ptl
+++ b/font-src/glyphs/number/4.ptl
@@ -10,7 +10,7 @@ glyph-block Digits-Four : begin
 	glyph-block-import Digits-Shared : OnumMarks ShiftDown CodeLnum CodeOnum
 
 	define [FourStdShape] : with-params [top open crossing [fine : AdviceStroke 3] [sw Stroke] [bbd 0]] : glyph-proc
-		local yBar : top * 0.3 + 0.625 * sw
+		local yBar : top * 0.3333 + 0.625 * sw
 
 		define xVertBar : [mix SB RightSB : if crossing 0.75 0.9125] - (bbd * 0.75) + [if crossing [HSwToV : 0.375 * sw] 0]
 		define yVertBarTop : [mix (yBar - sw) top : if open 0.5 1] - [if open (0.3 * sw) 0]
