diff --git a/packages/font-glyphs/src/number/4.ptl b/packages/font-glyphs/src/number/4.ptl
index aae08cc..0b002de 100644
--- a/packages/font-glyphs/src/number/4.ptl
+++ b/packages/font-glyphs/src/number/4.ptl
@@ -11,7 +11,7 @@ glyph-block Digits-Four : begin
 	glyph-block-import Digits-Shared : OnumMarks ShiftDown CodeLnum CodeOnum
 
 	define [FourStdShape] : with-params [top open crossing [fine [AdviceStroke 3]] [sw Stroke] [bbd 0] [slab SLAB]] : glyph-proc
-		define yBar : Math.max (top * 0.25 + 0.625 * sw) (top * 0.05 + sw + [if (!bbd && slab) Stroke 0])
+		define yBar : Math.max (top * 0.33 + 0.625 * sw) (top * 0.05 + sw + [if (!bbd && slab) Stroke 0])
 		define swVBar : [AdviceStroke 2.5] * (sw / Stroke)
 
 		define xVertBar : [mix SB RightSB : if crossing 0.78 0.9125] - (bbd * 0.75) + [if crossing [HSwToV : 0.375 * sw] 0]
